content/en/blog/spos-panel-at-kubecon-paris-2024.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "SPOS Panel at KubeCon Paris 2024"
+date: 2024-03-26T16:53:13+01:00
+author: Mauro Morales ([Twitter](https://twitter.com/mauromrls)) ([GitHub](https://github.com/mauromorales))
+---
+
+I recently had the opportunity to attend KubeCon 2024. You can find my recap at the [Spectro Cloud Blog](https://www.spectrocloud.com/blog/kubecon-paris-edge-ai-and-la-vie-en-cloud-native), but I'd like to add some additional information about the Special Purpose Operating System Panel in which we participated.
+
+The panel was schedule for the last day of KubeCon, which I think probably hurt the attendance since many people had already left. Nevertheless, according to the scheduling tool 351 people were planning to attend.
```

Comment:
```suggestion
The panel was scheduled for the last day of KubeCon, which I think probably hurt the attendance since many people had already left. Nevertheless, according to the scheduling tool, 351 people were planning to attend.
```